### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by tarunsingh5**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,24 +19,24 @@
...
       <div class="scores-container">
         <div class="score-container">0</div>
         <div class="best-container">0</div>
       </div>
     </div>
 
     <div class="above-game">
-      <p class="game-intro">Join the numbers and get to the <strong><span class="gurmukhi">2048</span> tile!</strong></p>
-      <a class="restart-button">New Game</a>
+      <p class="game-intro">ਨੰਬਰ ਮਿਲਾਓ ਅਤੇ ਨੰਬਰ ਮਿਲਾਓ ਅਤੇ <strong><span class="gurmukhi">2048</span>!</strong> ਟਾਈਲ ਪ੍ਰਾਪਤ ਕਰੋ </p>
```
Comment (navdeepsinghkhalsa) [inline index.html]: Double-paste

Comment (tarunsingh5) [reply index.html]: fixed

File: index.html [hunk 2]
```diff
@@ -73,7 +73,7 @@ <h1 class="title">2048</h1>
     </div>
 
     <p class="game-explanation">
-      <strong class="important">How to play:</strong> Use your <strong>arrow keys</strong> to move the tiles. When two tiles with the same number touch, they <strong>merge into one!</strong>
+      <strong class="important">ਕਿਵੇਂ ਖੇਡਣਾ ਹੈ: </ strong> ਟਾਈਲਾਂ ਨੂੰ ਮੂਵ ਕਰਨ ਲਈ ਆਪਣੇ <strong> ਐਰੋ ਬਟਨ </ strong> ਦੀ ਵਰਤੋਂ ਕਰੋ. ਜਦੋਂ ਇਕੋ ਨੰਬਰ ਦੇ ਨਾਲ ਦੋ ਟਾਈਲਾਂ ਛੋਹ ਜਾਂਦੀਆਂ ਹਨ, ਤਾਂ ਉਹ <strong> ਇੱਕ ਵਿੱਚ ਲੀਨ ਹੋ ਜਾਂਦੀਆਂ ਹਨ </ strong>
```
Comment (navdeepsinghkhalsa) [inline index.html]: The closing `</strong>` tags shouldn't have spaces

Comment (tarunsingh5) [reply index.html]: updated

File: index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>2048</title>
+  <title><span class="gurmukhi">2048</span></title>
```
Comment (navdeepsinghkhalsa) [inline index.html]: Unfortunately, this wouldn't work, as `<title>` only renders plain text. You would have to use unicode: ੨੦੪੮

Comment (tarunsingh5) [reply index.html]: updated

File: index.html [hunk 4]
```diff
@@ -19,24 +19,24 @@
 <body>
   <div class="container">
     <div class="heading">
-      <h1 class="title">2048</h1>
+      <h1 class="title"><strong><span class="gurmukhi">2048</span>!</strong></h1>
```
Comment (navdeepsinghkhalsa) [inline index.html]: The CSS styles already make the title bold and use Gurmukhi font, so these tags aren't needed :)

**Review by navdeepsinghkhalsa — APPROVED** [review]

**Pull request merged**